alphadia/outputtransform.py

```diff
@@ -956,31 +960,69 @@ def _build_run_stat_df(
             "proteins": channel_df["pg"].nunique(),
         }
 
-        if "weighted_mass_error" in channel_df.columns:
-            base_dict["ms1_accuracy"] = np.mean(channel_df["weighted_mass_error"])
-
         if "cycle_fwhm" in channel_df.columns:
             base_dict["fwhm_rt"] = np.mean(channel_df["cycle_fwhm"])
 
         if "mobility_fwhm" in channel_df.columns:
             base_dict["fwhm_mobility"] = np.mean(channel_df["mobility_fwhm"])
 
+        # collect optimization stats
+        base_dict["optimization.ms2_error"] = np.nan
```

Comment:
this might be a good time to start introducing centralized string constants, like e.g. https://github.com/MannLabs/alphapeptstats/blob/refactor_analysis_VIII/alphastats/keys.py

benefits: 
- no typos when having constants shared across multiple classes
- easy navigation between occurrences of a constant